src/theme/dojo/checkbox-group.m.css

```diff
@@ -0,0 +1,9 @@
+@import './variables.css';
+
+.root {
+	padding-top: var(--spacing-regular);
+	padding-bottom: var(--spacing-regular);
+}
+
+.legend {
```

Comment:
do you think this should perhaps compose the `label` styles?

Reply:
Yeah, not a bad shout